tests/test_ironoak.py

```diff
@@ -1,6 +1,17 @@
 
 from ironoak.cli import main
+from ironoak.person import persondetector
+import unittest
 
-
-def test_main():
-    assert main([]) == 0
+class TestMain(unittest.TestCase):
+    def test_person(self):
+        try:
+            result = False
+            det = persondetector.persondetector()
```

Comment:
Would suggest changing this to input an image or a video instead of a camera feed. This is because we want the test to run in limited time as a sanity check for any future changes. That way we know our new changes don't break the API.